views/js/portableLib/OAT/util/tooltip.js

```diff
@@ -29,12 +29,26 @@ define([
             $container.find('[data-role="tooltip-target"]').each(function(){
                 var $target = $(this),
                     $content,
+                    contentHtml,
                     contentId = $target.attr('aria-describedBy'),
-                    contentHtml;
+                    qtipPositionTarget = 'event';
 
                 if (contentId) {
                     $content = $container.find('#' + contentId);
                     if ($content.length) {
+                        var targetHeight = parseInt($($target.get(0)).css('height'), 10),
+                            targetFontSize = parseInt($($target.get(0)).css('fontSize'), 10);
```

Comment:
It is needed to unwrap and wrap again the element? `$target` should gives the same as `$($target.get(0))`.

Moreover, you should use the API instead of hacking it. So, to access the first element of a jQuery collection please do `$target.get(0)` instead of `$target[0]`.

Reply:
Removed the excessive wrapping.